src/main/java/com/saucelabs/jenkins/pipeline/SauceConnectStep.java

```diff
@@ -269,7 +269,7 @@ public boolean start() throws Exception {
             if (useGeneratedTunnelIdentifier) {
                 final String tunnelIdentifier = SauceEnvironmentUtil.generateTunnelIdentifier(job.getName());
                 overrides.put(SauceOnDemandBuildWrapper.TUNNEL_IDENTIFIER, tunnelIdentifier);
-                options = options + " --tunnel-identifier " + tunnelIdentifier;
+                options = options + " --tunnel-name " + tunnelIdentifier;
```

Comment:
you prob should `s/tunnelIdentifier/tunnelName` as well, since we're refactoring.

Reply:
Done in 7b1409f